Private/Utilities.ps1

```diff
@@ -34,17 +34,31 @@ url = https://github.com/3shapeAS/dockerbuild-pwsh.git
     $configData | Out-File -FilePath "$dotGitPath/config" -Encoding ascii
 }
 
-function Add-Postfix {
+function Add-RegistryPostfix {
     param (
-        [Parameter(Mandatory=$true)]
-        [String] $Data,
+        [String] $Registry,
         [ValidateNotNullOrEmpty()]
         [String] $Postfix = '/'
     )
 
-    $trimmedData = $Data.Trim()
-    if ( -Not $trimmedData.EndsWith($Postfix) ) {
-            $trimmedData += $Postfix
+    # Do nothing if $Data is empty / writespace
+    if ([String]::IsNullOrWhiteSpace($Registry)) {
+        return ''
+    }
+
+    $trimmedRegistry = $Registry.Trim()
+    if ( -Not $trimmedRegistry.EndsWith($Postfix) ) {
+            $trimmedRegistry += $Postfix
     }
-    $trimmedData
+    $trimmedRegistry
+}
+
+function Test-DockerDigest {
```

Comment:
Consider extracting to a separate file so this function can reside in the Private folder.

Reply:
👍